src/Workspaces/Core/Portable/Workspace/Solution/Project.cs

```diff
@@ -101,9 +101,11 @@ internal Project(Solution solution, ProjectState projectState)
         /// </summary>
         public HostLanguageServices LanguageServices => _projectState.LanguageServices;
```

Comment:
>LanguageServices [](http://example.com/codeflow?start=36&length=16)

Make this one obsolete? It's rather confusing to have both LanguageServices and Services.

Reply:
This will be easier to do once https://github.com/dotnet/roslyn/pull/63245 goes in.  I will do that in that PR as that's still a big use of this existing API.

Reply:
Done in 3008126bc97